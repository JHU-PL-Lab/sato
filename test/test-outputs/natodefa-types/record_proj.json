[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "r" ],
          "err_match_val": "{a = 2, b = 3}",
          "err_match_expected": "Record {c}",
          "err_match_actual": "Record {a, b}"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "r.c",
    "err_steps": 175
  }
]