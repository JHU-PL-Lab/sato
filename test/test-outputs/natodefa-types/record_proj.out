Type errors for:
- Input sequence : []
- Found at expr  : r.c
--------------------
* Value    : r = {a = 2, b = 3}
* Expected : Record {c}
* Actual   : Record {a, b}
1 errors found. No further control flows exist.
